src/components/CreditCard/CreditCard.jsx

```diff
@@ -0,0 +1,25 @@
+function CreditCard(props) {
+    const {type, number, expirationMonth, expirationYear, bank, owner, bgColor, color} = props;
+    
+     /* let visaCard () {
+        if(type === 'Visa') {
+            <img src='../../assets/images/visa.png'/>
+          } else {
+            <img src='../../assets/images/master-card.svg'/>
+           }
+     }
+     */
+      return (
+        <div>
+         <div style={{backgroundColor:`${bgColor}`, color:`${color}`}}>
+    
+    
+          <p>{number}</p>  
```

Comment:
On the credit card you should have **** followed by only the last 4 numbers of the card